java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/SampleDelegateCommandHandler.java

```diff
@@ -135,7 +130,7 @@ private static List<SymbolInformation> search(String projectName, String query,
 			targetProjects= ProjectUtils.getJavaProjects();
 		}
 
-		int s = IJavaSearchScope.REFERENCED_PROJECTS | IJavaSearchScope.SOURCES | IJavaSearchScope.APPLICATION_LIBRARIES;
+		int s = IJavaSearchScope.SOURCES | IJavaSearchScope.REFERENCED_PROJECTS | IJavaSearchScope.APPLICATION_LIBRARIES;
```

Comment:
does this make a big difference? Kind of scary

Reply:
it makes a minor difference, but it does reduce the search matches that are in the literal language. this reduction I think is fine.